src/mmunif.c

```diff
@@ -369,92 +365,92 @@ char unify(
       }
     }
   }
-  /* Add dummy token to end of schemeA and schemeB */
-  /* Use one beyond the last mathTokenArray entry for this */
+  // Add dummy token to end of schemeA and schemeB.
+  // Use one beyond the last mathTokenArray entry for this.
   nmbrLet(&schA, nmbrAddElement(schemeA, g_mathTokens));
   nmbrLet(&schB, nmbrAddElement(schemeB, g_mathTokens));
 
-  /* Initialize the usage of constants as the first, last,
-     only constant in a $a statement - for rejecting some simple impossible
-     substitutions - Speed-up: this is now done once and never deallocated */
-  /* g_firstConst is now cleared in eraseSource.c() (mmcmds.c)
-     to trigger this initialization after "erase" */
+  // Initialize the usage of constants as the first, last,
+  // only constant in a $a statement - for rejecting some simple impossible
+  // substitutions - Speed-up: this is now done once and never deallocated.
+  // g_firstConst is now cleared in eraseSource.c() (mmcmds.c)
```

Comment:
```suggestion
  //
  // g_firstConst is now cleared in eraseSource.c() (mmcmds.c)
```
I'm not sure how often to make this suggestion, but there does seem to be some loss of information here where multiple "paragraphs" are running together with the new comment style. Perhaps newlines like this should be used when it would be confusing.

Reply:
I wondered the same issue while making the changes. My solution was to split comments using periods, but I can see that this might not be enough in some circumstances. I think it's better if I continue with my simple strategy and then you suggest spacings when you think it's appropriate.